### PR TITLE
phantomjs. Add ssl params support 

### PR DESCRIPTION
**Pull request by Fabryprog**
ssl params support using functions:
- ignoreSslErrors
- sslCertificatesPath
- sslProtocol

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -105,8 +151,19 @@ Screenshot.prototype.capture = function(fn) {
   var opts = {
     maxBuffer: Infinity
   };
+  
+  var params = '';
+  if(this._ignoreSslErrors === true) {
+  	params = params + ' --ignore-ssl-errors=true ';
+  }
+  if(this._sslCertificatesPath != null && this._sslCertificatesPath != '') {
```
Comment (juliangruber) [inline index.js:158]: `if (this._sslCertificatesPath) {` checks whether it's a non zero length string, that would be enough

Comment (juliangruber) [inline index.js:158]: because `'' == false`

File: index.js [hunk 2]
```diff
@@ -105,8 +151,19 @@ Screenshot.prototype.capture = function(fn) {
...
+  	params = params + ' --ssl-certificates-path=' + this._sslCertificatesPath + ' ';
+  }
+  if(this._sslProtocol != null && this._sslProtocol != '') {
+  	params = params + ' --ssl-protocol=' + this._sslProtocol + ' ';
+  }
 
-  exec('phantomjs ' + args.join(' '), opts, function (err, stdout) {
+  exec('phantomjs ' + params + ' ' + args.join(' '), opts, function (err, stdout) {
```
Comment (juliangruber) [inline index.js]: those params should just be pushed onto the args array

File: index.js (hunk 1 above)
Comment (Fabryprog) [inline index.js:158]: It is strongest check than "zero length string"

Comment (juliangruber) [inline index.js:158]: yeah, but code clarity decreases more than stability increases

File: index.js (hunk 2 above)
Comment (Fabryprog) [inline index.js]: DONE and tested!

**Comment by juliangruber** [comment]
publish as 0.5.0, with minor changes

**Pull request merged**